index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Survey Form</title>
+</head>
+
+<body>
+  <header>
+    <h1>Survey Form</h1>
+    <p>Please enjoy this sick survey form</p>
+    <h6>(I worked really hard on it)</h6>
```

Comment:
I think an `h6` is probably not the right choice here. This text is not more important than the paragraph text above it, in fact it probably has equal importance. With that in mind, it should probably just be a paragraph tag that you style differently than the main text (for example, if you want it smaller or a different color, etc.)